notebooks/resource-allocation/generate-allocation-summary.py

```diff
@@ -0,0 +1,107 @@
+
+from qiita_db.software import Software
+from subprocess import check_output
+import pandas as pd
+from io import StringIO
+from qiita_db.exceptions import QiitaDBUnknownIDError
+from qiita_db.artifact import Artifact
+from qiita_db.analysis import Analysis
+from qiita_db.processing_job import ProcessingJob
+from qiita_db.metadata_template.prep_template import PrepTemplate
+from qiita_db.metadata_template.sample_template import SampleTemplate
+from json import loads
+from datetime import datetime
+
+all_commands = [c for s in Software.iter(False) for c in s.commands]
+main_jobs = [j for c in all_commands for j in c.processing_jobs
+             if j.status == 'success' and j.external_id.isnumeric()]
+
+sacct = ['sacct', '-p', '--format=JobID,ElapsedRaw,MaxRSS,Submit,Start,MaxRSS,'
+         'CPUTimeRAW,ReqMem,AllocCPUs,AveVMSize', '-j']
+
+data = []
+for i, j in enumerate(main_jobs):
+    if i % 1000 == 0:
+        print(f'{i}/{len(main_jobs)}')
+    eid = j.external_id
+    extra_info = ''
+    rvals = StringIO(check_output(sacct + [eid]).decode('ascii'))
+    _d = pd.read_csv(rvals, sep='|')
+    _d['QiitaID'] = j.id
+    cmd = j.command
+    s = j.command.software
+    try:
+        samples, columns, input_size = j.shape
+    except QiitaDBUnknownIDError:
+        continue
```

Comment:
I think it might be good to add a comment on why it's okay to ignore the QiitaDBUnknownIDError and continue, although I believe I understand why.